libvast/src/system/index.cpp

```diff
@@ -569,9 +569,9 @@ void index_state::schedule_lookups() {
                  it != persisted_partitions.end())
           part = inmem_partitions.get_or_load(partition_id);
       }
-      if (!part)
-        VAST_VERBOSE("{} failed to load partition {} that was part of a query",
-                     *self, partition_id);
+      if (!part && !next->erased)
```

Comment:
Can we skip the loading attempt directly above if the partition is marked erased?